Dockerfile

```diff
@@ -39,8 +38,6 @@ ENV NODE_OPTIONS="--max-http-header-size=81920 --dns-result-order=ipv4first"
 
 
 ENV NODE_ENV production
-# Uncomment the following line in case you want to disable telemetry during runtime.
-ENV NEXT_TELEMETRY_DISABLED 1
```

Comment:
Why is this removed?

Reply:
Not sure why it got removed. Will add it back.